cypress/e2e/tests/migration/dynamic-report/issues/filter.sorting.pagination.test.ts

```diff
@@ -95,7 +95,7 @@ describe(["@tier3"], "Filtering, sorting and pagination in Issues", function ()
     });
 
     it("All issues - Filtering issues by name", function () {
-        Analysis.analyzeAll(applicationsList[0]);
+        Analysis.analyzeAll(applicationsList[7]);
```

Comment:
Why such a specific number? maybe should we add a comment there explaining why its 7 instead of 0 or 6

Reply:
It can be 6 or more, as there are 12 applications with index from 0 to 11, starting from 6 - analysis for daytrader app is performed which means more than 10 issues on the page, if number will be from 0 to 5 - only 10 issues on page means pagination test will fail

Reply:
That should be specified in a comment as it is very specific and can be missed by people not familiar with that test suite.

Reply:
Comment added